### PR TITLE
docs: add differences between Chip, Tag, & Badge

### PR DESCRIPTION
**Pull request by snitin315**

### CODE REVIEW

File: packages/blade/src/components/Badge/_decisions/decisions.md [hunk 1]
```diff
@@ -1,71 +1,80 @@
...
-  variant='positive'
-  contrast='high'
-  icon={InfoIcon} 
-  size='large' 
-  fontWeight='bold'
->
+<Badge variant="positive" contrast="high" icon={InfoIcon} size="large" fontWeight="bold">
   New
-</Badge>
+</Badge>;
```
Comment (snitin315) [inline packages/blade/src/components/Badge/_decisions/decisions.md:29]:
```suggestion
</Badge>
```

File: packages/blade/src/components/Badge/_decisions/decisions.md [hunk 2]
```diff
@@ -1,71 +1,80 @@
...
-| Prop | Type | Default | Description | Required |
-|---|---|---|---|---|
-| children | `string` | `undefined` | The label to be rendered within the badge. | ✅ |
-| variant | `positive`,`negative`, `notice`, `information`, `neutral` | `neutral` | The variant of the badge to be rendered. |  |
-| contrast | `low`, `high` | `low` | The contrast of the badge to be rendered. |  |
-| size | `medium`, `small` | `medium` | The size of the badge to be rendered. |  |
-| icon | `Icon` | `undefined` | The Blade `Icon` component to be rendered for the badge. |  |
-| fontWeight | `bold`, `regular` | `regular` | The font weight of the badge's font. |  |
+| Prop       | Type                                                      | Default     | Description                                              | Required |
+| ---------- | --------------------------------------------------------- | ----------- | -------------------------------------------------------- | -------- |
```
Comment (snitin315) [inline packages/blade/src/components/Badge/_decisions/decisions.md:14]: This file got formatted.

**Review by anuraghazra — APPROVED** [review]

File: packages/blade/src/components/Badge/_decisions/decisions.md [hunk 3]
```diff
@@ -1,71 +1,80 @@
...
-  icon={InfoIcon} 
-  size='large' 
-  fontWeight='bold'
->
+<Badge variant="positive" contrast="high" icon={InfoIcon} size="large" fontWeight="bold">
   New
 </Badge>
 ```
 
+## Badge Differentiation
```
Comment (kamleshchandnani) [inline packages/blade/src/components/Badge/_decisions/decisions.md:32]:
```suggestion
## Difference between `Badge` vs `Tag` vs `Chip`
```

File: packages/blade/src/components/Tag/_decisions/decisions.md [hunk 4]
```diff
@@ -174,6 +174,10 @@ We can extend our `value` prop to accept JSX
 
 </details>
 
+## Tag Differentiation
```
Comment (kamleshchandnani) [inline packages/blade/src/components/Tag/_decisions/decisions.md:177]:
```suggestion
## Difference between `Badge` vs `Tag` vs `Chip`
```

Comment (kamleshchandnani) [reply packages/blade/src/components/Tag/_decisions/decisions.md:177]: @snitin315 don't see this applied?

Comment (snitin315) [reply packages/blade/src/components/Tag/_decisions/decisions.md]: Bug

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**